gnd/src/main/java/com/google/android/gnd/rx/BooleanOrError.java

```diff
@@ -19,21 +19,21 @@
 import javax.annotation.Nullable;
 
 /** The result of an operation that can return either true, false, or fail with an exception. */
-public class BooleanResult extends ValueOrError<Boolean> {
-  private BooleanResult(@Nullable Boolean value, @Nullable Throwable error) {
+public class BooleanOrError extends ValueOrError<Boolean> {
```

Comment:
It's a real shame java doesn't have type synonyms!

Reply:
Indeed!